spec/systems/users/view_sign_in_page_spec.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'View sign in page', type: :system do
+  context 'given an unathenticated user' do
```

Comment:
Typo:

```suggestion
  context 'given an unauthenticated user' do
```

Reply:
Fixed in ac6eaf2feef7c5ac21a171dd692a4e7d84bc32b3